react-ui/src/Components/Initiatives.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../assets/Styles/styles.css';
+
+class Initiatives extends React.Component {
```

Comment:
this could also be titled Projects
but if already discussed with team am fine with it